<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinMyInfo</class>
 <widget class="QDialog" name="WinMyInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WinMyInfo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,QPushButton,QToolButton
{font: 12pt &quot;Adobe 黑体 Std R&quot;;}</string>
  </property>
  <widget class="HeadPor" name="frame">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>20</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCHead">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>46</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/固定/Resources/固定/登录按钮tip.jpg);</string>
   </property>
   <property name="text">
    <string>更换头像</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>322</x>
     <y>461</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame_2{image: url(:/用户界面/Resources/固定/我的信息背景.jpg);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>密保问题</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>密保答案</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbQuestion">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <item>
     <property name="text">
      <string>不设置密保</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>你的真实姓名是？</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>你的生日是？</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>你父亲的名字是？</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>你母亲的名字是？</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>你最喜欢的明星是？</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>你最好的朋友是？</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineAnswer">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>200</width>
      <height>35</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>密保答案</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/固定/Resources/固定/登录按钮tip.jpg);</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>btnCHead</zorder>
  <zorder>btnCancel</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HeadPor</class>
   <extends>QFrame</extends>
   <header location="global">headpor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../BMS.qrc"/>
 </resources>
 <connections/>
</ui>
